src/Nox.EntityFramework.Postgres/PostgresMarkdownDatabaseConfiguration.cs

```diff
@@ -0,0 +1,14 @@
+using Nox.Types;
+using Nox.Types.EntityFramework.Types;
+
+namespace Nox.EntityFramework.Postgres;
+
+public class PostgresMarkdownDatabaseConfiguration : MarkdownDatabaseConfigurator
```

Comment:
Since this is a base string type, I copied what is being done for Text and this is currently Text only has specific configuration for Postgres, exactly like I'm doing here.